index.html

```diff
@@ -115,9 +115,9 @@ <h1 class="section-header">About Speedometer 2.1</h1>
 
                         <p>
                             This benchmark simulates user actions for adding, completing, and removing to-do items using multiple examples in TodoMVC. Each example in TodoMVC implements the same todo application using DOM APIs in different ways. Some
-                            call DOM APIs directly from ECMAScript 5 (ES5), ECMASCript 2015 (ES6), ES6 transpiled to ES5, and Elm transpiled to ES5. Others use one of eleven popular JavaScript frameworks: React, React with Redux, Ember.js,
-                            Backbone.js, AngularJS, (new) Angular, Vue.js, jQuery, Preact, Inferno, and Flight. Many of these frameworks are used on the most popular websites in the world, such as Facebook and Twitter. The performance of these types
-                            of operations depends on the speed of the DOM APIs, the JavaScript engine, CSS style resolution, layout, and other technologies.
+                            call DOM APIs directly from ECMAScript 5 (ES5), ECMASCript 2015 (ES6), ES6 transpiled to ES5, and Elm transpiled to ES5. Others use one of twelve popular JavaScript frameworks: React, React with Redux, Ember.js,
+                            Backbone.js, AngularJS, (new) Angular, Vue.js, jQuery, Preact, Inferno, Flight, and Lit. Many of these frameworks are used on the most popular websites in the world, such as Facebook and Twitter. The performance of these
```

Comment:
We got rid of old angular, inferno, and flight. If we're updating this list, we might as delete those as well.

Reply:
There's more text that should change, since we have different type of workloads now. 
Maybe we should revert this change and tackle an update separately?

Reply:
Yeah, that would make sense.

Reply:
Reverted.

Reply:
Thanks - changes regarding the about text can be tracked in this issue: https://github.com/WebKit/Speedometer/issues/165